doc/mma.md

```diff
@@ -1,9 +1,11 @@
 # MMA Features
 
-We are beginning to add multi-messenger astronomy (MMA) features to the app. In particular, we are ingesting events from [GCN](https://gcn.gsfc.nasa.gov/); the service controlling this is here: https://github.com/skyportal/skyportal/blob/master/skyportal/services/gcn_service/gcn_service.py.
+Skyportal supports multi-messanger astronomy (MMA) features. In particular, we ingest events from [GCN](https://gcn.gsfc.nasa.gov/); the service controlling this is here: https://github.com/skyportal/skyportal/blob/master/skyportal/services/gcn_service/gcn_service.py.
```

Comment:
I think `messenger` was the right spelling...

Reply:
HA!